content/en/references/configuration.md

```diff
@@ -304,10 +304,11 @@ To learn more about these configuration options, see [DNS Server]({{< ref "dns-s
 
 | Variable | Example Values | Description |
 | - | - | - |
-| `DNS_ADDRESS` | 0.0.0.0 (default) | Address the LocalStack should bind the DNS server on (port 53 tcp/udp). Value `0` to disable.
-| `DNS_SERVER` | 8.8.8.8 (default) | Fallback DNS server for non-modified queries.
-| `DNS_RESOLVE_IP` | 127.0.0.1 | IP address the DNS integration should return as A record for modified queries. This will override any automatic detection of the proper response IP.
-| `DNS_LOCAL_NAME_PATTERNS` | | Names which should be resolved to the LocalStack IP, as python-compatible regex.
+| `DISABLE_TRANSPARENT_ENDPOINT_INJECTION` | `0` (default in Pro) \| `1` | Whether to disable DNS resolution of AWS hostnames to the LocalStack container. Pro feature. (see [Transparent Endpoint Injection]({{< ref "user-guide/tools/transparent-endpoint-injection" >}}))
+| `DNS_ADDRESS` | `0.0.0.0` (default) | Address the LocalStack should bind the DNS server on (port 53 tcp/udp). Value `0` to disable.
+| `DNS_SERVER` | `8.8.8.8` (default) | Fallback DNS server for non-modified queries.
+| `DNS_RESOLVE_IP` | `127.0.0.1` | IP address the DNS integration should return as A record for modified queries. This will override any automatic detection of the proper response IP.
```

Comment:
I adjusted "for queries handled by LocalStack" accordingly

Reply:
I think this is a confusing artifact of the default value having meaning, rather than being blank. The actual behaviour is to return `DNS_RESOLVE_IP` if `DNS_RESOLVE_IP != 127.0.0.1`, and use the DNS server's logic to look up the value if not (i.e. its internal mappings or from an upstream). This is a bit strange, since the value returned may not be `127.0.0.1` (e.g. when querying `localhost.localstack.cloud` from a correctly configured docker container or lambda function). Hence why I used "is customised".

Reply:
very good point. Thanks 👍 
(this dynamic behavior is still not covered well and people might be surprised why it works from within a Lambda container)

sadly badly confusing 😕

Reply:
re-added the "customized" part (us-en) ;)